### DISCUSSION TITLE
web interface timeout

### DISCUSSION THREAD

**Discussion by alanlubold** [gravitational/teleport#9461] [Q&A]
How is the logout timeout configured on the web interface of Teleport, versions 4 or 5? I find that I am logged out after a relatively short period of inactivity, not from the sessions with nodes, but from the web interface itself. 

Thanks

**Comment by Valien** [comment]
This might help - https://github.com/gravitational/teleport/blob/ab857001ded16aa5315af759ae20eed947c05b52/lib/defaults/defaults.go#L120
